### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata @notwaldorf PTAL

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+      Alice in Wonderland
+    </div>
+
+    <paper-dialog-scrollable class="flex paper-font-body1">
+
+        <p>Alice was beginning to get very tired of sitting by her sister
+      on the bank, and of having nothing to do: once or twice she had
+      peeped into the book her sister was reading, but it had no
+      pictures or conversations in it, 'and what is the use of a book,'
+      thought Alice 'without pictures or conversation?'</p>
```
Comment (cdata) [inline demo/index.html:58]: Nice ipsum :100:

File: demo/index.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+      <p>She ate a little bit, and said anxiously to herself, 'Which
+      way? Which way?', holding her hand on the top of her head to feel
+      which way it was growing, and she was quite surprised to find
+      that she remained the same size: to be sure, this generally
+      happens when one eats cake, but Alice had got so much into the
+      way of expecting nothing but out-of-the-way things to happen,
+      that it seemed quite dull and stupid for life to go on in the
+      common way.</p>
+
+      <p>So she set to work, and very soon finished off the cake.</p>
```
Comment (cdata) [inline demo/index.html:279]: :cake:

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "paper-dialog-scrollable",
+  "version": "0.8.0",
+  "authors": "The Polymer Authors",
```
Comment (cdata) [inline bower.json:4]: Nit, I think this is an array in most places.

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "git://github.com/PolymerElements/paper-dialog-scrollable"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/PolymerElements/paper-dialog-scrollable",
+  "ignore": [],
+  "dependencies": {
+    "polymer": "Polymer/polymer#v0.8.0-rc.7"
+  },
+  "devDependencies": {
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
```
Comment (cdata) [inline bower.json:23]: `iron-component-page` is here to stay :cry:

File: demo/index.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
+<!doctype html>
+<!--
+  @license
+  Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+  This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+  The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+  The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+  Code distributed by Google as part of the polymer project is also
+  subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
```
Comment (cdata) [inline demo/index.html]: Indentation :abcd:

File: index.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+  <title>paper-dialog-scrollable</title>
+
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../polymer/polymer.html">
+  <link rel="import" href="../iron-doc-viewer/iron-doc-viewer.html">
```
Comment (cdata) [inline index.html:22]: `<core-component-page>`

File: bower.json (hunk 4 above)
Comment (morethanreal) [inline bower.json:23]: Can we mass-change later? :(

**Comment by morethanreal** [comment]
Ok, fixed the indent. Does it LGTY?

File: paper-dialog-scrollable.html [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+      // FIXME use resizeable
+      this.async(this._scroll, 100);
+    },
+
+    _scroll: function() {
+      if (this.scrollTarget.scrollTop > 0) {
+        this.classList.add('is-scrolled');
+      } else {
+        this.classList.remove('is-scrolled');
+      }
```
Comment (cdata) [inline paper-dialog-scrollable.html:109]: I don't have a good solution, but I wish that there was a cleaner way toggle all of these classes.

File: paper-dialog-scrollable.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-flex-layout/classes/iron-flex-layout.html">
+<link rel="import" href="../paper-styles/paper-styles.html">
+
+<!--
+`paper-dialog-scrollable` implements a scrolling area used in a Material Design dialog. Use this
```
Comment (cdata) [inline paper-dialog-scrollable.html:16]: I'm kind of curious: why is this useful over a plain container with `overflow: auto` on it? I'm just interested in the use case, and I don't understand what the "dividers" are.

File: bower.json (hunk 4 above)
Comment (cdata) [inline bower.json:23]: WFM

**Comment by cdata** [comment]
LGTM, plus a casual inquiry :+1:

File: paper-dialog-scrollable.html (hunk 8 above)
Comment (morethanreal) [inline paper-dialog-scrollable.html:16]: It hides and shows a divider (just a line) at the top or the bottom based on scroll position to indicate the presence of scrolling content.

File: paper-dialog-scrollable.html (hunk 7 above)
Comment (morethanreal) [inline paper-dialog-scrollable.html:109]: Me too :(

**Pull request merged**